src/mbgl/style/expression/assertion.cpp

```diff
@@ -35,6 +35,23 @@ ParseResult Assertion::parse(const Convertible& value, ParsingContext& ctx) {
     return ParseResult(std::make_unique<Assertion>(it->second, std::move(parsed)));
 }
 
+const char* Assertion::getOperator() const {
+    // TODO: Should we strip assertions entirely from serialization?
```

Comment:
We should leave them -- as of now, we don't track whether an assertion was automatically inferred or actually written as part of the expression.  If the latter, it could easily affect the expression's meaning.